Classes/MapEditor/MapEditor.cs

```diff
@@ -355,6 +355,29 @@ public class MapEditor {
         }
         AddNotes(notes);
     }
+
+    public void QuantizeSelection() {
+        if (currentMapDifficulty == null) {
+            return;
+        }
+
+        double beats = 60.0 / (globalBPM * 2);
```

Comment:
If I understand correctly, this will always quantize to half a beat - shouldn't we consider the beat division setting in here?

Please note that this would be either coming from global setting or from timing change, depending on where the notes are located.

Reply:
I guess taking timing changes into account might be complicated, as with the BPM changes the beats will not be aligned to global beat. 

I think it's fair to take just the global beat division into account, although this way it will work weirdly with BPM changes.

Reply:
If you want to implement it correctly for BPM changes as well, take a look at StepManiaMapConverter class, where I had to implement logic to calculate this when converting SM notation to global beats.

You could refactor code responsible for doing the global beat to local beat translations into separate class and reuse it here.

Reply:
this part will quantize to one quarter beat. 
but you are correct it make sence to consider the beat division of the setting. i will have a look if i find some time to do that.